server/cluster/cluster.go

```diff
@@ -429,6 +429,19 @@ func (c *RaftCluster) runStoreConfigSync() {
 			if err := c.opt.Persist(c.GetStorage()); err != nil {
 				log.Warn("store config persisted failed", zap.Error(err))
 			}
+			// If the cluster was set up with `raft-kv2` engine, this cluster should
+			// open `evict-slow-trend` scheduler as default.
+			{
+				name := schedulers.EvictSlowTrendType
+				args := []string{}
+
+				s, err := schedulers.CreateScheduler(name, c.GetOperatorController(), c.GetStorage(), schedulers.ConfigSliceDecoder(name, args), c.GetCoordinator().GetSchedulersController().RemoveScheduler)
```

Comment:
How about letting coordinator to create the scheduler?

Reply:
FYI, maybe u means https://github.com/tikv/pd/blob/346e7716e2598dfba2db6afa73c3731e15449f49/server/cluster/cluster.go#L783-L784 ?
It's succinct enough for it.

Reply:
As discussed off-line, this part has been polished. And the test results show that it's valid.
![image](https://github.com/tikv/pd/assets/18441614/d8da71bb-7303-4e84-82bf-fea5b98b93e5)